### PR TITLE
add system performance and description

### PR DESCRIPTION
**Pull request by xixi019**
based on https://arxiv.org/pdf/2303.10368.pdf

### CODE REVIEW

**Comment by xixi019** [comment]
#135

File: freebase/freebaseQA.md [hunk 1]
```diff
@@ -33,6 +50,7 @@ This dataset can be downloaded via the [link](https://github.com/kelvin-jiang/Fr
 |    T5-base+C4    | 2022 |    28.33    |    -     |  -  |   EN     |   [Moiseev et al.](https://arxiv.org/pdf/2205.08184.pdf)    |
 |   T5-base+KELM   | 2022 |    28.15    |    -     |   -  |  EN     |   [Moiseev et al.](https://arxiv.org/pdf/2205.08184.pdf)    |
 |     T5-base      | 2022 |    27.55    |    -     |   -  |  EN     |   [Moiseev et al.](https://arxiv.org/pdf/2205.08184.pdf)    |
+|  KGQA-RR(GOT2)| 2023 |     - |  5.09   |  -  |   EN     |  [Hu et al.](https://arxiv.org/pdf/2303.10368.pdf)   |
```
Comment (DavidBlavid) [inline freebase/freebaseQA.md]: Typo -> rename GOT2 to GPT2

**Comment by DavidBlavid** [comment]
found only one typo, looks good otherwise 👍

**Comment by xixi019** [comment]
> found only one typo, looks good otherwise 👍

fixed

**Pull request closed without merging**